alyssa_mckee/Django/belt_reviwer/apps/book_app/models.py

```diff
@@ -0,0 +1,67 @@
+from django.db import models
+import re
+
+# Create your models here.
+class AuthorManager(models.Manager):
+	def validate_author(self, data):
+		errors = []
+		#new_author takes presidense over existing
+		
+		#if no new specified and no author chosen:
+		print(not data['new_author'] or not data['existing_author'])
+		if data['new_author']=="" and data['existing_author']=="":
+			errors.append("Author can not be empty")
+		
+		#if new author is not empty and is not alpha (with spaces between)
+		if data['new_author'] != "" and not re.match(r'^[a-zA-Z]+([a-zA-Z ]?[a-zA-Z]+)*$',data['new_author']):
```

Comment:
Nice job on this regex, especially since it considers names that are more than a first name and last name. ex: Sir Arthur Conan Doyle. +1